client/src/Components/ProductList/ProductList.css

```diff
@@ -0,0 +1,3 @@
+.product-card-container {
+    padding: 5px 5px 5px 5px;
```

Comment:
You can just use `padding: 5px;` when the padding will be the same on every side.